wremnants/datasets/datagroups.py

```diff
@@ -242,11 +247,10 @@ def setHists(self, baseName, syst, procsToRead=None, label=None, nominalIfMissin
         # but this would need to assume that fakes effectively had all the single processes in each group as members
         # (usually it will be the case, but it is more difficult to handle in a fully general way and without bugs)
         histForFake = None # to store the data-MC sums used for the fakes, for each syst
-        nameFake = "Fake" # TODO: actual name might/should be configurable
-        if nameFake in procsToRead:
-            procsToReadSort = [x for x in procsToRead if x != nameFake] + [nameFake]
+        if sumFakesPartial and self.fakeName in procsToRead:
```

Comment:
I don't think I understand what "sumFakesPartial" means

Reply:
It's to disable the feature that fakes are created "on the fly" when reading the other processes, to restore the original behaviour where the fakes are treated as any other group but everything is read again. This is needed for the ABCD method to use the same name i.e. "Fake" for the nonprompt background group, which was a previous comment.

Reply:
Not worth rerunning the CI just for this but the name is pretty confusing

Reply:
Not sure... but I don't know what makes it partial.

Reply:
partial because when looping over the process groups in each step of the loop the process is counted in parallel for the fakes (since each step is only a part of the fakes, partial)